Assignment-4/src/Team-member.js

```diff
@@ -0,0 +1,84 @@
+const TeamData = [

```

Comment:
We can have a practice to make it a json file.
It makes it easier for us to write test cases in future.
Its a good practice.